dist/angular-editable-text.js

```diff
@@ -26,13 +26,12 @@
         onChange: '&ggOnChange'
       },
       transclude: true,
```

Comment:
This can be removed, since the directive template no longer includes ng-transclude.